### ISSUE TITLE
Live Graph - No Data.

### ISSUE THREAD

**Issue by sillihkram** [NagiosEnterprises/ncpa#432]
Live data is not displaying on graph.

OS Version info:
```
[user@mgmt-vm0715 ~]$ cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.4 (Maipo)

[user@mgmt-vm0715 ~]$ uname -r
3.10.0-693.1.1.el7.x86_64
```

NCPA Version Info:
```
[user@mgmt-vm0715 ~]$ sudo yum info ncpa
Loaded plugins: enabled_repos_upload, package_upload, product-id, search-disabled-repos, subscription-
              : manager
Installed Packages
Name        : ncpa
Arch        : x86_64
Version     : 2.1.1
Release     : 1.el7.centos
Size        : 20 M
Repo        : installed
From repo   : nagios-base
Summary     : A cross-platform active and passive monitoring agent
URL         : https://www.nagios.org/ncpa/help.php
License     : Nagios Open Software License Version 1.3
Description : The Nagios Cross-Platform Agent is used with Nagios XI and Nagios Core to run active
            : and/or passive checks on any operating system. Installs with zero requirements using a
            : bundled version of Python.

Uploading Enabled Repositories Report
Loaded plugins: product-id
```

Service status:
```
[user@mgmt-vm0715 ~]$ sudo systemctl status ncpa_listener 
● ncpa_listener.service - LSB: This manages the NCPA Listener service
   Loaded: loaded (/etc/rc.d/init.d/ncpa_listener; bad; vendor preset: disabled)
   Active: active (running) since Tue 2018-02-06 23:31:42 CST; 21min ago
     Docs: man:systemd-sysv-generator(8)
  Process: 31431 ExecStop=/etc/rc.d/init.d/ncpa_listener stop (code=exited, status=0/SUCCESS)
  Process: 31464 ExecStart=/etc/rc.d/init.d/ncpa_listener start (code=exited, status=0/SUCCESS)
   CGroup: /system.slice/ncpa_listener.service
           └─31475 /usr/local/ncpa/ncpa_listener --start

Feb 06 23:31:42 mgmt-vm0715.ecmsvc.com systemd[1]: Starting LSB: This manages the NCPA Listener service...
Feb 06 23:31:42 mgmt-vm0715.ecmsvc.com ncpa_listener[31464]: Starting NCPA Listener: [  OK  ]
Feb 06 23:31:42 mgmt-vm0715.ecmsvc.com systemd[1]: Started LSB: This manages the NCPA Listener service.
```

Log Information:
```
[user@mgmt-vm0715 ~]$ tailf /usr/local/ncpa/var/log/ncpa_listener.log
2018-02-06 23:40:24,549 31475 INFO 10.171.24.155 - - [2018-02-06 23:40:24] "GET /api/processes/?token=n%40gi05C0r3&warning=450&critical=500&check=1 HTTP/1.1" 200 15387 0.104756
2018-02-06 23:40:58,413 31475 INFO 10.171.24.155 - - [2018-02-06 23:40:58] "GET /api/memory/virtual/?token=n%40gi05C0r3&warning=90&critical=99&units=G&check=1 HTTP/1.1" 200 369 0.007189
2018-02-06 23:41:55,090 31475 INFO 10.171.24.155 - - [2018-02-06 23:41:55] "GET /api/disk/logical/%7Cvar%7Clib%7Cpulp/used_percent/?token=n%40gi05C0r3&warning=80&critical=95&check=1 HTTP/1.1" 200 242 0.006568
2018-02-06 23:42:10,992 31475 INFO 10.171.24.155 - - [2018-02-06 23:42:10] "GET /api/cpu/percent/?token=n%40gi05C0r3&warning=20&critical=40&check=1&aggregate=avg HTTP/1.1" 200 229 0.506705
2018-02-06 23:43:37,162 31475 INFO 10.171.24.155 - - [2018-02-06 23:43:37] "GET /api/disk/logical/%7C/used_percent/?token=n%40gi05C0r3&warning=80&critical=95&check=1 HTTP/1.1" 200 242 0.005976
2018-02-06 23:45:15,778 31475 INFO 10.171.24.155 - - [2018-02-06 23:45:15] "GET /api/service/sshd/running/?token=n%40gi05C0r3&check=1 HTTP/1.1" 200 276 0.004364
2018-02-06 23:45:24,708 31475 INFO 10.171.24.155 - - [2018-02-06 23:45:24] "GET /api/processes/?token=n%40gi05C0r3&warning=450&critical=500&check=1 HTTP/1.1" 200 15321 0.099601
2018-02-06 23:45:58,492 31475 INFO 10.171.24.155 - - [2018-02-06 23:45:58] "GET /api/memory/virtual/?token=n%40gi05C0r3&warning=90&critical=99&units=G&check=1 HTTP/1.1" 200 369 0.007692
2018-02-06 23:46:55,155 31475 INFO 10.171.24.155 - - [2018-02-06 23:46:55] "GET /api/disk/logical/%7Cvar%7Clib%7Cpulp/used_percent/?token=n%40gi05C0r3&warning=80&critical=95&check=1 HTTP/1.1" 200 242 0.006525
2018-02-06 23:47:10,557 31475 INFO 10.171.24.155 - - [2018-02-06 23:47:10] "GET /api/cpu/percent/?token=n%40gi05C0r3&warning=20&critical=40&check=1&aggregate=avg HTTP/1.1" 200 229 0.507050
2018-02-06 23:48:29,019 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:29] "GET /gui/stats HTTP/1.1" 200 6983 0.001431
2018-02-06 23:48:29,725 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:29] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 125412 0.438284
2018-02-06 23:48:29,849 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:29] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 31423 0.001736
2018-02-06 23:48:29,899 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:29] "GET /static/css/ncpa.css HTTP/1.1" 200 9750 0.001383
2018-02-06 23:48:29,995 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:29] "GET /static/css/ncpa-graph.css HTTP/1.1" 200 924 0.001350
2018-02-06 23:48:30,322 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:30] "GET /static/js/smoothie.js HTTP/1.1" 200 32440 0.001684
2018-02-06 23:48:30,601 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:30] "GET /graph/memory/virtual/percent HTTP/1.1" 200 5500 0.017861
2018-02-06 23:48:30,606 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:30] "GET /api/interface/ HTTP/1.1" 200 765 0.004277
2018-02-06 23:48:30,614 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:30] "GET /api/disk/logical HTTP/1.1" 200 3041 0.007565
2018-02-06 23:48:30,619 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:30] "GET /graph/cpu/percent HTTP/1.1" 200 5445 0.004513
2018-02-06 23:48:31,601 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/interface/lo/bytes_sent?delta=1&units=K HTTP/1.1" 200 5526 0.005796
2018-02-06 23:48:31,605 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /api/memory/virtual/percent HTTP/1.1" 200 279 0.003647
2018-02-06 23:48:31,610 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/interface/lo/bytes_recv?delta=1&units=K HTTP/1.1" 200 5526 0.005117
2018-02-06 23:48:31,615 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/interface/eth0/bytes_sent?delta=1&units=K HTTP/1.1" 200 5549 0.004669
2018-02-06 23:48:31,793 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/interface/eth0/bytes_recv?delta=1&units=K HTTP/1.1" 200 5549 0.004809
2018-02-06 23:48:31,803 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/disk/logical/%7Cboot%7Cefi/used_percent HTTP/1.1" 200 5565 0.004642
2018-02-06 23:48:31,809 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/disk/logical/%7Cdev%7Chugepages/used_percent HTTP/1.1" 200 5577 0.005177
2018-02-06 23:48:31,814 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:31] "GET /graph/disk/logical/%7Csys%7Cfs%7Cselinux/used_percent HTTP/1.1" 200 5595 0.004619
2018-02-06 23:48:32,000 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7Cproc%7Cfs%7Cnfsd/used_percent HTTP/1.1" 200 5598 0.005088
2018-02-06 23:48:32,032 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7Cvar%7Clib%7Cmongodb/used_percent HTTP/1.1" 200 5613 0.005651
2018-02-06 23:48:32,044 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7C/used_percent HTTP/1.1" 200 5538 0.005335
2018-02-06 23:48:32,049 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7Csys%7Ckernel%7Cconfig/used_percent HTTP/1.1" 200 5623 0.004677
2018-02-06 23:48:32,233 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/interface/lo/bytes_sent?delta=1&units=K HTTP/1.1" 200 285 0.003991
2018-02-06 23:48:32,237 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/interface/eth0/bytes_sent?delta=1&units=K HTTP/1.1" 200 285 0.003745
2018-02-06 23:48:32,246 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7Cboot/used_percent HTTP/1.1" 200 5558 0.004873
2018-02-06 23:48:32,410 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/interface/eth0/bytes_recv?delta=1&units=K HTTP/1.1" 200 285 0.003841
2018-02-06 23:48:32,501 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/interface/lo/bytes_recv?delta=1&units=K HTTP/1.1" 200 285 0.003711
2018-02-06 23:48:32,510 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /graph/disk/logical/%7Cvar%7Clib%7Cpulp/used_percent HTTP/1.1" 200 5585 0.004601
2018-02-06 23:48:32,653 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cdev%7Chugepages/used_percent HTTP/1.1" 200 283 0.003517
2018-02-06 23:48:32,706 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/cpu/percent HTTP/1.1" 200 295 0.505143
2018-02-06 23:48:32,711 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cproc%7Cfs%7Cnfsd/used_percent HTTP/1.1" 200 283 0.003473
2018-02-06 23:48:32,724 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cvar%7Clib%7Cmongodb/used_percent HTTP/1.1" 200 283 0.003475
2018-02-06 23:48:32,851 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7C/used_percent HTTP/1.1" 200 284 0.003490
2018-02-06 23:48:32,920 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cboot%7Cefi/used_percent HTTP/1.1" 200 283 0.003456
2018-02-06 23:48:32,924 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cboot/used_percent HTTP/1.1" 200 284 0.003382
2018-02-06 23:48:32,942 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:32] "GET /api/disk/logical/%7Cvar%7Clib%7Cpulp/used_percent HTTP/1.1" 200 284 0.003542
2018-02-06 23:48:33,076 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:33] "GET /api/disk/logical/%7Csys%7Ckernel%7Cconfig/used_percent HTTP/1.1" 200 283 0.004387
2018-02-06 23:48:33,130 31475 INFO 10.1.17.141 - - [2018-02-06 23:48:33] "GET /api/disk/logical/%7Csys%7Cfs%7Cselinux/used_percent HTTP/1.1" 200 283 0.003782
2018-02-06 23:48:37,226 31475 INFO 10.171.24.155 - - [2018-02-06 23:48:37] "GET /api/disk/logical/%7C/used_percent/?token=n%40gi05C0r3&warning=80&critical=95&check=1 HTTP/1.1" 200 242 0.005732

```

Browser Info:
Chrome Version 63.0.3239.132 (Official Build) (64-bit):
![screen shot 2018-02-07 at 12 42 52 am](https://user-images.githubusercontent.com/19679959/35900680-387cf262-0ba1-11e8-96c0-b75c309b4a37.png)

Browser info:
Safari Version 11.0.3 (13604.5.6)
![screen shot 2018-02-07 at 1 04 10 am](https://user-images.githubusercontent.com/19679959/35902061-ce31eeb0-0ba7-11e8-995e-3e979512147a.png)

Browser info:
Firefox ESR 52.3.0 (64-bit)
![screen shot 2018-02-07 at 1 37 57 am](https://user-images.githubusercontent.com/19679959/35902071-d8638ae2-0ba7-11e8-92ce-ddccfadd3b35.png)

Actual perf data:

```
Tasks: 226 total,   2 running, 224 sleeping,   0 stopped,   0 zombie
%Cpu(s): 24.4 us,  1.8 sy,  0.0 ni, 73.7 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
KiB Mem :  3850112 total,   128796 free,  2200820 used,  1520496 buff/cache
KiB Swap:  5242876 total,  5165304 free,    77572 used.  1332976 avail Mem 

   PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                              
  2820 apache    20   0 1307244 113328   3460 R 100.0  2.9   5:48.48 /usr/bin/python /usr/bin/celery worker -n reserved_resource_worker-3@%h -A pulp.serv+
  2463 root      20   0  573236  17960   4420 S   2.0  0.5   8:57.54 /opt/BESClient/bin/BESClient                                                         
     9 root      20   0       0      0      0 S   0.3  0.0   0:02.73 [rcu_sched]                                                                          
  1329 qpidd     20   0  498384  17884   1900 S   0.3  0.5   0:10.73 /usr/sbin/qpidd --config /etc/qpid/qpidd.conf                                        
  1344 apache    20   0  669180  27468   2472 S   0.3  0.7   0:23.46 /usr/bin/python /usr/bin/celery beat --app=pulp.server.async.celery_instance.celery +
  1491 apache    20   0  666812  54300   2616 S   0.3  1.4   0:06.79 /usr/bin/python /usr/bin/celery worker -n reserved_resource_worker-1@%h -A pulp.serv+
  1719 apache    20   0 1167952  76608   2896 S   0.3  2.0   0:16.25 (wsgi:pulp)     -DFOREGROUND                                                         
  1725 apache    20   0 1233488  78588   2932 S   0.3  2.0   0:16.34 (wsgi:pulp)     -DFOREGROUND                                                         
 33754 mhillis   20   0  168180   2376   1572 R   0.3  0.1   0:00.10 top                                                                                  
     1 root      20   0  195580   4128   1236 S   0.0  0.1   0:05.72 /sbin/init                                                                           
     2 root      20   0       0      0      0 S   0.0  0.0   0:00.05 [kthreadd]                                                                           
     3 root      20   0       0      0      0 S   0.0  0.0   0:00.03 [ksoftirqd/0]                                                                        
     5 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [kworker/0:0H]                                                                       
     7 root      rt   0       0      0      0 S   0.0  0.0   0:00.05 [migration/0]                                                                        
     8 root      20   0       0      0      0 S   0.0  0.0   0:00.00 [rcu_bh]                                                                             
    10 root      rt   0       0      0      0 S   0.0  0.0   0:00.04 [watchdog/0]                                                                         
    11 root      rt   0       0      0      0 S   0.0  0.0   0:00.03 [watchdog/1]                                                                         
    12 root      rt   0       0      0      0 S   0.0  0.0   0:00.02 [migration/1]                                                                        
    13 root      20   0       0      0      0 S   0.0  0.0   0:00.04 [ksoftirqd/1]                                                                        
    15 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [kworker/1:0H]                                                                       
    16 root      rt   0       0      0      0 S   0.0  0.0   0:00.03 [watchdog/2]                                                                         
    17 root      rt   0       0      0      0 S   0.0  0.0   0:00.02 [migration/2]                                                                        
    18 root      20   0       0      0      0 S   0.0  0.0   0:00.04 [ksoftirqd/2]                                                                        
    20 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [kworker/2:0H]                                                                       
    21 root      rt   0       0      0      0 S   0.0  0.0   0:00.04 [watchdog/3]                                                                         
    22 root      rt   0       0      0      0 S   0.0  0.0   0:00.03 [migration/3]                                                                        
    23 root      20   0       0      0      0 S   0.0  0.0   0:00.04 [ksoftirqd/3]                                                                        
    25 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [kworker/3:0H]                                                                       
    27 root      20   0       0      0      0 S   0.0  0.0   0:00.00 [kdevtmpfs]                                                                          
    28 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [netns]                                                                              
    29 root      20   0       0      0      0 S   0.0  0.0   0:00.00 [khungtaskd]                                                                         
    30 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [writeback]                                                                          
    31 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [kintegrityd]                                                                        
    32 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 [bioset]  
```

**Comment by jomann09** [comment]
This should be fixed in 2.1.2 - it's a common issue that they aren't working in 2.1.1 - hopefully the new version will be out shortly, just a couple things to test and do before then.

**Comment by jomann09** [comment]
Duplicate of #419
